crypto/onetimesig.go

```diff
@@ -42,6 +42,7 @@ type OneTimeSignature struct {
 
 	// Old-style signature that does not use proper domain separation.
 	// PKSigOld is a signature of (PK || BatchID) under the master key (OneTimeSignatureVerifier).
+	// TODO(adam): This should be safe to delete now, right?
```

Comment:
Probably no, for a stupid reason: this struct was never annotated with `omitempty`, so certificates contain votes with `ps` set to an all-zero `ed25519Signature`.  Removing this field will cause those certificate sigs to no longer decode, because there's no `ps`.

Reply:
Should we start requiring `PKSigOld` to be zero, or should we just ignore it?